docs/docs/routing.md

```diff
@@ -2,6 +2,10 @@
 title: Routing
 ---
 
+Part of what makes Gatsby sites so fast is that a lot of the work is done at build time and the running site is using mostly [static content](/docs/adding-app-and-website-functionality/#static-pages). Specifically, each page is its own HTML file that's generated at build time. During that process, Gatsby creates paths to access those files, handling routing for you. Navigating in a Gatsby app requires understanding what those paths are and how they're generated.
```

Comment:
Perhaps we want to clarify that we only load the HTML file for the initial page the user visits?

Reply:
Ya, I went back and forth on this one. I don't want to get too far into the implementation and confuse users. The text still reads true for me, since we create all the files and generate the paths, even if we don't use all of them in that manner.
Perhaps removing the entire "Specifically," sentence and just mentioning that Gatsby creates paths at build time?